### DISCUSSION TITLE
What are the advantages of switching to using net6?

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#9079] [Q&A]
_[Question from Discord.](https://discord.com/channels/372137812037730304/550416151172087808/988363440563294278)_

**Comment by jeromelaban** [comment]
If the questions are specific to mobile, .NET 6 offers the new SDK-style project format, which makes it easier to work with recent builds of .NET. This includes all the new APIs of .NET 6, C# 10, faster builds, more recent tooling for single projects.

It also includes the newer Mac Catalyst (UIKit based) target for running macOS apps instead of AppKit based apps.

**Comment by pkar70** [comment]
It also makes your app unavailable for phones (Windows based). Yes, I know, Microsoft ends support for their phones.

**Comment by mterwoord** [comment]
Great info. Also any benefits for targetting linux (Gtk)? Or not much?

**Reply by jeromelaban** [reply]
On Linux, it's mostly about the .NET 6 itself, and everything it brings. You'll find that in [Microsoft's documentation for .NET 6](https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-6).
